README.md

```diff
@@ -1,2 +1,57 @@
 # census-field-service-cucumber
 Cucumber acceptance tests for the Census Integration teams Field Service
+
+Before you can run all the cucumber features in this project you need to set it up, locally, to point at the relevant runtime environment (either LOCAL, DEV_LOCAL, or DEV). The following steps need to be done:
+
+# The Census RH Service uses the RUNTIME_ENV environment variable to determine the credentials and URLs to use in the scenarios. 
+
+In the census-field-service-cucumber repo set it to either LOCAL or DEV_LOCAL or DEV:
```

Comment:
I've got no idea what the difference is between LOCAL and DEV-LOCAL. If this is a common project then I guess the reader should know what they mean, but otherwise this readme should explain it.

Reply:
Good point - I have now added explanations of LOCAL, DEV_LOCAL, and DEV, to the readme.